arcgis-ios-sdk-samples/Display information/Display grid/DisplayGridSettingsViewController.swift

```diff
@@ -151,42 +172,66 @@ class DisplayGridSettingsViewController: UITableViewController {
         grid.labelVisibility = priorGrid.labelVisibility
         grid.isVisible = priorGrid.isVisible
         
-        if let selectedColor = gridColorPicker.selectedColor {
-            changeGridColor(selectedColor)
+        if let priorGridColor = gridColor(of: priorGrid) {
```

Comment:
I'm confused by the use of `prior` in the variable name when that seems to be what is going to be the new color.  Or maybe I have that the wrong way around.  I think with the changes you've made here `prior` no longer applies.